go/build-and-test/action.yml

```diff
@@ -56,13 +56,13 @@ runs:
       run: echo "short_sha=`echo ${GITHUB_SHA::7}`" > $GITHUB_OUTPUT
       shell: bash
     - name: Configure Go environment
-      uses: wishabi/github-actions/go/configure@v0
+      uses: wishabi/github-actions/go/configure@asdf-support
```

Comment:
I'll convert this back before merging, but wanted to get alignment on the approach.

It's working with [this](https://github.com/wishabi/distribution-api/actions/runs/8423243921/job/23104952892) version of distribution-api that was converted to asdf.